src/main/java/ugh/fileformats/mets/MetsMods.java

```diff
@@ -112,18 +108,48 @@
 import ugh.dl.MetadataType;
 import ugh.dl.Person;
 import ugh.dl.Prefs;
-import ugh.dl.Reference;
 import ugh.dl.VirtualFileGroup;
-import ugh.exceptions.DocStructHasNoTypeException;
 import ugh.exceptions.ImportException;
-import ugh.exceptions.MetadataTypeNotAllowedException;
 import ugh.exceptions.MissingModsMappingException;
-import ugh.exceptions.PreferencesException;
-import ugh.exceptions.ReadException;
-import ugh.exceptions.TypeNotAllowedAsChildException;
-import ugh.exceptions.TypeNotAllowedForParentException;
-import ugh.exceptions.UGHException;
-import ugh.exceptions.WriteException;
+
+/*******************************************************************************
```

Comment:
Why are you adding the license header again?

Reply:
Eclipse treats the license header as comment of an import, when it is misplaced below the package declaration, which was the case in this file. this is the original license header which was “sorted” here by Eclipse. Since the imports are collapsed by default, I didn’t see that it was moved down here, neither did one of my reviewers. I got the review comment why I had “deleted” the license header, which which was not so true. So I added one on top of the file. The one on top is the new one, this one is the original one. I will fix this. This response is only for your understanding.